<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于xray启动器</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item alignment="Qt::AlignTop">
    <widget class="QLabel" name="logo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icons/app_icon.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightPanel" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel{
  min-height: 23px;
}
QPushButton{
  min-height: 32px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="labelTitle">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>xray启动器 0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelBuild">
        <property name="text">
         <string>本软件基于Qt xxx 构建</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>构建于 yyyy-mm-dd</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>项目地址</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;a href=&quot;https://github.com/liuguangw/XrayLauncher&quot;&gt;https://github.com/liuguangw/XrayLauncher&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>162</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
